### PR TITLE
Move comment to right spot

### PR DESCRIPTION
**Pull request by RHavar**
...and kill some extraneous whitespace

### CODE REVIEW

File: src/wallet/wallet.cpp [hunk 1]
```diff
@@ -2145,11 +2145,11 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     setCoinsRet.clear();
     nValueRet = 0;
 
-    // List of values less than target
+
     pair<CAmount, pair<const CWalletTx*,unsigned int> > coinLowestLarger;
     coinLowestLarger.first = std::numeric_limits<CAmount>::max();
     coinLowestLarger.second.first = NULL;
-    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue;
+    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
```
Comment (dcousens) [inline src/wallet/wallet.cpp]: `std::`

File: src/wallet/wallet.cpp [hunk 2]
```diff
@@ -2145,11 +2145,11 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     setCoinsRet.clear();
     nValueRet = 0;
 
-    // List of values less than target
+
     pair<CAmount, pair<const CWalletTx*,unsigned int> > coinLowestLarger;
     coinLowestLarger.first = std::numeric_limits<CAmount>::max();
     coinLowestLarger.second.first = NULL;
-    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue;
+    std::vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
```
Comment (benma) [inline src/wallet/wallet.cpp]: I think the previous comment location was correct, as it is not only about this line, but all the lines that follow (the BOOST_FOREACH block below, which populates the vector). The few lines above is also needed for that.

Adding `std::` only here is a bit pointless, the whole file uses vector, set, pair etc. unqualified. Maybe a separate PR could explicitly qualify all of them, if preferred. Personally, I like it.

File: src/wallet/wallet.cpp (hunk 1 above)
Comment (benma) [inline src/wallet/wallet.cpp]: I think the previous comment location was correct, as it is not only about this line, but all the lines that follow (the BOOST_FOREACH block below, which populates the vector). The few lines above is also needed for that.

Adding `std::` only here is a bit pointless, the whole file uses vector, set, pair etc. unqualified. Even the `pair` on the same line is not qualified.
Maybe a separate PR could explicitly qualify all of them, if preferred.  Personally, I would like it.

File: src/wallet/wallet.cpp [hunk 3]
```diff
@@ -2149,7 +2149,7 @@ bool CWallet::SelectCoinsMinConf(const CAmount& nTargetValue, const int nConfMin
     pair<CAmount, pair<const CWalletTx*,unsigned int> > coinLowestLarger;
     coinLowestLarger.first = std::numeric_limits<CAmount>::max();
     coinLowestLarger.second.first = NULL;
-    vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
+    std::vector<pair<CAmount, pair<const CWalletTx*,unsigned int> > > vValue; // List of values less than target
```
Comment (laanwj) [inline src/wallet/wallet.cpp]: This is already done (for the entire file) by @kallewoof in https://github.com/bitcoin/bitcoin/pull/9643

**Pull request closed without merging**